pkg/scheduler/internal/cache/cache.go

```diff
@@ -259,6 +263,16 @@ func (cache *cacheImpl) UpdateSnapshot(nodeSnapshot *Snapshot) error {
 		cache.updateNodeInfoSnapshotList(nodeSnapshot, updateAllLists)
 	}
 
+	if updateUsedPVCSet {
```

Comment:
Can we put this in `updateNodeInfoSnapshotList`?  No need to iterate over nodes multiple times.
https://github.com/yibozhuang/kubernetes/blob/7feaacd27ad18830ad8d299f6078d120526665f0/pkg/scheduler/internal/cache/cache.go#L292